src/AuthContext.tsx

```diff
@@ -34,9 +36,32 @@ export function useAuth(): AuthContextData {
 export function AuthProvider({ children }: { children: React.ReactNode }) {
   const [currentUser, setCurrentUser] = useState<User | null>(null);
   const [isLoading, setIsLoading] = useState(true);
+  const [isStaff, setIsStaff] = useState<Boolean | null>(null);
```

Comment:
for context, how can isStaff be null rather than being true or false? This seems confusing